Sources/AloeStackView/AloeStackView.swift

```diff
@@ -551,16 +574,20 @@ open class AloeStackView: UIScrollView {
     if animated {
       cell.alpha = 0
       layoutIfNeeded()
-      UIView.animate(withDuration: 0.3) {
+      UIView.animate(withDuration: 0.3, animations: {
         cell.alpha = 1
+      }) { flag in
```

Comment:
This is a little confusing, perhaps we could split this handler out to a separate `completionHandler` above, like with the `removeCell` method?

Reply:
remove block closure, just input argument